### ISSUE TITLE
'Import from twitch user' feature doesn't get all follows

### ISSUE THREAD

**Issue by azizLIGHT** [maweki/twitchlive-extension#72]
Not all followed streamers are captured by the 'import from twitch user' feature. Some are there and some are not. I cannot tell why a streamer is added to the list and some are not added. I checked this against the website and tried the process repeatedly for my twitch user and got the same results

**Comment by maweki** [comment]
It would be helpful if you could link to one ore more streamers that were not imported. We had some issues with screen names vs. account names and spaces in names. Last time my test imports all seemed to work but from the code I didn't see how for the channels where the channel name differs from the account name.

**Comment by azizLIGHT** [comment]
Using my twitch username azizLIGHT, here's a sample of a few streamers that were not added that are currently live at the moment: asmongold, lirik, botezlive, clintstevens, reckful

**Comment by maweki** [comment]
Do you follow more than 100 people?

**Comment by azizLIGHT** [comment]
At least over 200 I would say!

**Comment by maweki** [comment]
Oh, okay. I have to look this up but it could be that we only get the first 100. Would you guess that about half of the streamers are missing? I don't think that it has something to do with their usernames or whether they are online or not.

Also, could you try adding one of the missing ones by hand, just to check that this works?

**Comment by azizLIGHT** [comment]
FYI: I actually did manually add the ones I mentioned successfully and they show OK. 

I would say a lot are missing because I just checked with a browser extension for twitch and it shows 44 online for me right now whereas the gnome extension just shows 18

**Comment by maweki** [comment]
Yeah, that looks like about half. Thank you. I will look into this. I'd never thought somebody could follow that many streamers so I think we never implemented the bulk-querying. Twitch queries are limited to 100 streamers per query.
